projects/github-actions/required-review/src/requirement.js

```diff
@@ -52,32 +53,29 @@ function buildReviewerFilter( config, teamConfig, indent ) {
 	const op = keys[ 0 ];
 	let arg = teamConfig[ op ];
 
+	// Shared validation.
```

Comment:
Any reason we don't just do an `items.includes( op )` instead of a `switch`?

Reply:
Not really. The way I think about this code a `switch` just makes more sense to me.